.github/workflows/lint.yaml

```diff
@@ -2,7 +2,7 @@
 name: Linters
 on:  # yamllint disable-line rule:truthy
   pull_request:
-    branches: ['main']
+    branches: ['development']
```

Comment:
Shouldn't this be also the stage and prod branches? Or any branch really? It never hurts to run tests and linters for any PR, right? IOW, how about we just remove this line? (Same applies for task tests.)

Reply:
Some people (not me) create branches on the repo instead of using a fork. There is nothing wrong per say with running the checks against those branches, but it is kind of a waste. For staging and prod, I didn't add it because they are just getting the commits from development - if they were good enough to pass development, they should be good enough for the other branches too. I can remove it though I don't think the wasted resources is a big deal

Reply:
> Some people (not me) create branches on the repo instead of using a fork. There is nothing wrong per say with running the checks against those branches, but it is kind of a waste. 

To be clear, in this context we were only talking about workflows that run when you open a pull request - not simply when you push something to a random topic branch. And people typically don't create PRs against their own topic branches, no?

> if they were good enough to pass development, they should be good enough for the other branches too.

Yes, this is true. But I think it's the least we can do to ensure stuff doesn't get screwed up. To be honest I still don't fully understand exactly how the promotion will happen. You cannot simply create a PR with origin=development and target=staging, because this would break the default references. So there will need to be a middle topic branch. And at least initially this will be a manual process. So you can never be 100 % sure that everything is correct.